src/controllers/contact/contact.presenter.ts

```diff
@@ -0,0 +1,6 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ContactPresenter {
+  @ApiProperty()
+  readonly success: boolean;
+}
```

Comment:
コントローラーでContactPresenterインスタンス化するようにしたい
```suggestion
export class ContactPresenter {
  @ApiProperty()
  readonly success: boolean;
  
  constructor() {
    this.success = true;
  }
}
```